packages/injective-cosmwasm/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "injective-cosmwasm"
-version = "0.1.65"
+version = "0.1.66"
```

Comment:
we will need to merge with dev branch and publish a new package I think

Reply:
Okay already merged, but I assume we need to publish a merged package version still?